weather-app/src/App.tsx

```diff
@@ -1,9 +1,10 @@
 import './App.css';
+import CitiesList from './Components/CitiesList/CitiesList';
 
 function App() {
   return (
     <div className="App">
-      
+      <CitiesList/>
```

Comment:
We should place here not components but page

Reply:
Thank you for the comment. I will change that :)